addons/project/models/project.py

```diff
@@ -759,6 +760,11 @@ def project_update_all_action(self):
         action['display_name'] = _("%(name)s's Updates", name=self.name)
         return action
 
+    def action_project_sharing(self):
+        action = self.env['ir.actions.act_window']._for_xml_id('project.project_sharing_project_task_action')
+        action['display_name'] = _("%(name)s", name=self.name)
```

Comment:
In methods above, `_("%(name)s blabla", name=self.name)` is used to translate 'blabla'.
As there is no 'blabla' to translate here :
```suggestion
        action['display_name'] = self.name
```

Reply:
Yep, makes sense thanks